### PR TITLE
@ashfurrow => View in Room fixes for 6 and 6+ - fixes #199

### LINKED ISSUE #199: View in Room rotation mode offset incorrectly on 6 & 6+

### PR DESCRIPTION
**Pull request by orta**
Couldn't come up with a reasonable algorithm for the resizing, so had to do it per device ( it's pixel exact on a bunch of images. )

selfie-0

### CODE REVIEW

File: Artsy/Classes/Categories/Apple/UIDevice-Hardware.h [hunk 1]
```diff
@@ -69,7 +72,7 @@ typedef enum {
     UIDeviceUnknownAppleTV,
     UIDeviceIFPGA,
 
-} UIDevicePlatform;
+} ;
```
Comment (ashfurrow) [inline Artsy/Classes/Categories/Apple/UIDevice-Hardware.h]: Superfluous whitespace.

File: Artsy/Classes/Categories/Apple/UIDevice-Hardware.h [hunk 2]
```diff
@@ -34,7 +34,7 @@
 #define SIMULATOR_IPAD_NAMESTRING       @"iPad Simulator"
 #define SIMULATOR_APPLETV_NAMESTRING    @"Apple TV Simulator" // :)
 
-typedef enum {
+typedef NS_ENUM(NSUInteger, UIDevicePlatform) {
```
Comment (ashfurrow) [inline Artsy/Classes/Categories/Apple/UIDevice-Hardware.h]: We ought to do a run through to find old, non-`NS_ENUM` uses of enums.

**Comment by ashfurrow** [comment]
Looks good, modulo comments.

File: Artsy/Classes/Categories/Apple/UIDevice-Hardware.h (hunk 2 above)
Comment (orta) [inline Artsy/Classes/Categories/Apple/UIDevice-Hardware.h]: Done'd

**Pull request merged**